tools/db_bench_tool.cc

```diff
@@ -2363,6 +2416,15 @@ class CombinedStats {
     }
   }
 
+  double CalcStdDev(std::vector<double> data, double average) {
+    double squared_sum = 0.0;
+    for (double x : data) {
+      squared_sum += std::pow(x - average, 2);
+    }
+
+    return std::sqrt(squared_sum / data.size());
```

Comment:
I believe we need Bessel's correction here. https://en.wikipedia.org/wiki/Bessel%27s_correction (The small sample count intervals looked too small to me in one example you sent me.)